configure-fiddler/tasks/fiddler-preferences.md

```diff
@@ -0,0 +1,390 @@
+---
+title: Configurable Preferences
+description: List of Fiddler CLassic configurable preferences
+slug: fiddler-classic-preferences
+publish: true
+position: 30
+---
+
+# Preferences in Fiddler Classic
+
+Fiddler's preferences feature allows you to view and alter many configuration settings.
+
+The following is a partial list of supported preferences in Fiddler; the default value is shown when one exists.
+
+Shows error messages when Inspector and Extension related errors are encountered:
```

Comment:
reading this it seems I need to set showerrors to "False" in order to see the errors. Is this correct?